modules/codegen/src/smithy4s/codegen/internals/SmithyResources.scala

```diff
@@ -29,10 +29,16 @@ private[smithy4s] object SmithyResources {
 
   def produce(
       resourceOutputFolder: os.Path,
-      localSmithyFiles: List[os.Path],
+      specs: List[os.Path],
       namespaces: List[String]
   ): List[os.Path] = {
 
+    val localSmithyFiles = specs.flatMap { spec =>
+      if (os.isDir(spec)) os.walk(spec).filter(_.ext == "smithy")
```

Comment:
What about json files?

Reply:
oh wait, is this about the files generated in https://github.com/disneystreaming/smithy4s/pull/736 ?

Reply:
actually you have a point

Reply:
It is indeed about #736, but you are correct about the fact that this filter right here needs to allow for json files. This is addressed in [f25c1fa](https://github.com/disneystreaming/smithy4s/pull/789/commits/f25c1fa1fbf9ae108f59a9b8a4e242d42e5ad80e)